src/epidemik/MetaEpiModel.py

```diff
@@ -163,17 +163,24 @@ def travel_step(x, populations):
         # Travel occurs independently for each compartment
         # since we don't allow in-flight transitions
         for comp in compartments_.columns:
-            new_compartments[comp] = travel.apply(travel_step, populations=compartments_[comp]).sum(axis=1)
+            new_compartments[comp] = travel.apply(
+                    travel_step, 
+                    populations=compartments_[comp]
+            ).sum(axis=1)
             
         return new_compartments
     
-    def _run_spread(self):
+    def _run_spread(self) -> None:
         for state in self.compartments_.index:
             pop = self.compartments_.loc[state].to_dict()
             self.models[state].single_step(**pop)
             self.compartments_.loc[state] = self.models[state].values_.iloc[[-1]].values[0]
 
-    def simulate(self, timestamp, t_min=1, seasonality=None, seed_state=None, susceptible='S', **kwargs):
```

Comment:
seed_state should be a string. It essentially the name of the population that is being seeded,  and corresponds to the index of the population DataFrame